distribution/docs/src/main/resources/_contents/_sources/csw-federated-source-contents.adoc

```diff
@@ -0,0 +1,28 @@
+
+==== CSW Federated Source
+
+The CSW federated source supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
+
+===== Installing the CSW Federated Source
+
+The CSW Federated Source is | is not installed by default with a standard installation in the ${ddf-spatial} application.
```

Comment:
I'm not sure on this one. There is "CSW Specification Profile Federated Source" and "CSW Federation Profile Source" (which is below)